README.md

```diff
@@ -1,33 +1,44 @@
 # AppLovin MAX SDK
```

Comment:
I'd add a banner with our logos at the top and format center
```
[![MAX Logo](https://www.applovin.com/wp-content/uploads/2023/06/logo_home_products_max.svg)](https://applovin.com)
```
looks like this and is clickable
[![MAX Logo](https://www.applovin.com/wp-content/uploads/2023/06/logo_home_products_max.svg)](https://applovin.com)